### ISSUE TITLE
ModuleNotFoundError No module named numpy

### ISSUE THREAD

**Issue by User2587** [SciSharp/Keras.NET#44]
I'm getting this error when I was executing the below code:
//Load train data
            NDarray x = np.array(new float[,] { { 0, 0 }, { 0, 1 }, { 1, 0 }, { 1, 1 } });
            NDarray y = np.array(new float[] { 0, 1, 1, 0 });

            //Build sequential model
            var model = new Sequential();
            model.Add(new Dense(32, activation: "relu", input_shape: new Shape(2)));
            model.Add(new Dense(64, activation: "relu"));
            model.Add(new Dense(1, activation: "sigmoid"));

            //Compile and train
            model.Compile(optimizer: "sgd", loss: "binary_crossentropy", metrics: new string[] { "accuracy" });
            model.Fit(x, y, batch_size: 2, epochs: 1000, verbose: 1);

            //Save model and weights
            string json = model.ToJson();
            File.WriteAllText("model.json", json);
            model.SaveWeight("model.h5");

            //Load model and weight
            var loaded_model = Sequential.ModelFromJson(File.ReadAllText("model.json"));
            loaded_model.LoadWeight("model.h5");

numpy is already installed in 
C:\Users\MyName\AppData\Local\Programs\Python\Python36\Lib\site-packages

**Comment by deepakkumar1984** [comment]
What does this shows: 
import numpy
numpy.__version__

**Comment by User2587** [comment]
numpy version is 1.17.2

**Comment by deepakkumar1984** [comment]
Somehow the Visual Studio is unable to pick the python path from environment variable. This is something I cannot debug from code. Did you try restarting your machine?

**Comment by User2587** [comment]
What you said is right, I have written incorrect python path. Now its working! Sorry for the inconvenience.

**Comment by solarflarefx** [comment]
Can you describe where you changed the path?
